platform/src/test/kotlin/org/stellar/anchor/platform/custody/fireblocks/FireblocksPaymentServiceTest.kt

```diff
@@ -155,4 +156,25 @@ class FireblocksPaymentServiceTest {
 
     JSONAssert.assertEquals(responseJson, gson.toJson(response), JSONCompareMode.STRICT)
   }
+
+  @Test
+  fun test_getTransactionsByTimeRange_success() {
```

Comment:
You can also try updating this test to make two calls Fireblocks(to verify `while` piece of code)